packages/xo-web/src/common/intl/messages.js

```diff
@@ -12,7 +12,7 @@ const messages = {
   statusLoading: 'Loading…',
   errorPageNotFound: 'Page not found',
   errorNoSuchItem: 'no such item',
-  errorUnknownItem: 'unknown item',
+  errorUnknownItem: 'unknown {type}',
```

Comment:
When you break a message, don't forget to check if it should also be updated in any of the locales. (I don't think there's any in this case.)

Reply:
I already checked.